src/lib/Zikula/Bundle/FormExtensionBundle/Form/Type/LocaleType.php

```diff
@@ -29,13 +30,13 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
         ]);
     }
 
-    public function getName()
+    public function getBlockPrefix()
```

Comment:
I believe this change was introduced in Symfony 2.8. If that is the case here (or with other things you are changing) then they should be changed only in the Core-1.4 branch and then merged up to Core-2.0. This means that large monolithic changes like this PR (currently) are probably not the best way to handle this transition.

ANYTHING that will work with Symfony 2.8 must be first merged into Core-1.4 and then merged up.